src/applications/vaos/components/TabNav.jsx

```diff
@@ -1,35 +1,38 @@
 import React from 'react';
-import { withRouter } from 'react-router';
+import { useHistory, useLocation } from 'react-router-dom';
 import { focusElement } from 'platform/utilities/ui';
 
 import TabItem from './TabItem';
 
-export function TabNav({ location, router, hasExpressCareRequests }) {
+export default function TabNav({ hasExpressCareRequests }) {
+  const history = useHistory();
+  const location = useLocation();
+
   const isExpressCareTab = location.pathname === '/express-care';
   return (
     <ul className="va-tabs vaos-appts__tabs" role="tablist">
       <TabItem
         id="upcoming"
         tabpath="/"
-        isActive={location.pathname === '/'}
+        isActive={location.pathname.endsWith('appointments/')}
```

Comment:
This change still points to [the `rootUrl` for the app in the `registry.json` file](https://github.com/department-of-veterans-affairs/vets-website/blob/60d389a984c9b20641776fe730187b49d013acba/src/applications/registry.json#L55), correct?

Reply:
Yeah, I probably should use that here.